cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -404,30 +404,60 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             return parseInt(this.$('#due_in').val());
         },
 
+        showProjectedDate: function() {
+            var startDate = DateUtils.parseDateFromString(this.model.get('start'))
+            this.$("#startDate").html(startDate.toDateString()); // xss-lint: disable=javascript-jquery-html
```

Comment:
I agree with Matthew here that we should try and avoid these. Are you able to use `.val` or some other jQuery method to set the value inside this span? Right now, it doesn't like it because you are directly editing the HTML. It considers this a risk since if a bad actor was able to modify what you use here, they could write anything they want onto the page (i.e. malicious attacks)

That being said, you are also using multiple other libraries here which could be sanitizing the input and ensuring it is safe. You have the parseDateFromString and the toDateString methods which might not allow malicious code to pass through anyway. 

Let me know if you need help looking into this.